### DISCUSSION TITLE
Rule of thumb for headland size?

### DISCUSSION THREAD

**Discussion by giusetalarico** [Fields2Cover/Fields2Cover#29] [Q&A] [answered]
As subject: any ideas?
I see this in documentation but I think is kinda generic. What do you think?

`F2CCells no_hl = const_hl.generateHeadlands(cells, 3.0 * robot.robot_width);`

Thanks for any help.

**Comment by Gonzalo-Mier** [comment, marked as answer]
To choose the width of the headlands, farmers usually use a natural number multiplied by the width of the robot, or half of the spray boom width. The value of the width also depends on the turning radius of the robot. Big machines need more space on the headlands. More space on headlands means less space to harvest.

Right now, one of the master student of my research group is researching exactly on how to create those headland optimally according to a given objective function, but it will take some time yet.
